receiver/postgresqlreceiver/scraper.go

```diff
@@ -305,6 +305,9 @@ func (p *postgreSQLScraper) collectTopQuery(ctx context.Context, dbClient client
 				row[dbAttributePrefix+columnName] = finalValue
 			}
 		}
+		if row[dbAttributePrefix+totalExecTimeColumnName] == 0.0 {
```

Comment:
Am I understanding it correctly that value is a string here? this doesn't seem to be safe?